GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1551 @@
+                Console.WriteLine("Default Domain Name: " + defaultDomain.Name);
+
+                // Get Tenant's Tech Contacts
+                foreach (string techContact in tenantDetail.TechnicalNotificationMails)
+                {
+                    Console.WriteLine("Tenant Tech Contact: " + techContact);
+                }
+                return tenantDetail;
+            }
+
+            #endregion
+        }
+
+        private static async Task<User> GetSignedInUser(ActiveDirectoryClient client)
+        {
+            #region Get signed user info, get their photo, and update their photo
+
+            #region Get signed in user details
+
+            User signedInUser = new User();
+            try
+            {
+                signedInUser = (User) await client.Me.ExecuteAsync();
+                Console.WriteLine("\nUser UPN: {0}, DisplayName: {1}", signedInUser.UserPrincipalName,
+                    signedInUser.DisplayName);
+            }
+            catch (Exception e)
+            {
+                Program.WriteError("\nError getting signed in user {0}", Program.ExtractErrorMessage(e));
+            }
+
+
+            #endregion
+
+            #region get signed in user's photo
+
+            if (signedInUser.ObjectId != null)
+            {
+                IUser sUser = (IUser) signedInUser;
+                IStreamFetcher photo = (IStreamFetcher) sUser.ThumbnailPhoto;
+                try
+                {
+                    DataServiceStreamResponse response =
+                        await photo.DownloadAsync();
+                    Console.WriteLine("\nUser {0} GOT thumbnailphoto", signedInUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError getting the user's photo - may not exist {0}",
+                        Program.ExtractErrorMessage(e));
+                }
+            }
+            return signedInUser;
+
+            #endregion
+        }
+
+        private static async Task UpdateSignedInUsersPhoto(IUser signedInUser)
+        {
+            #region update signed in user's photo
+
+            // NOTE:  Updating the signed in user's photo requires User.ReadWrite (when available) or 
+            // Directory.ReadWrite.All or Directory.AccessAsUser.All
+            if (signedInUser.ObjectId != null)
+            {
+                Console.WriteLine("\nDo you want to update your thumbnail photo? yes/no\n");
+                string update = Console.ReadLine();
+
+                if (update != null && update.Equals("yes"))
+                {
+                    try
+                    {
+                        Console.WriteLine("\nSpecify path of photo:");
+                        string photo = Console.ReadLine();
+                        //TODO - update with allowed art and save locally with project

```

Comment:
removing for now. Maybe we'll have a default photo later

---
In reply to: [81239933](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r81239933) [](ancestors = 81239933)